src/firejail/main.c

```diff
@@ -1286,8 +1287,15 @@ int main(int argc, char **argv, char **envp) {
 		// filtering
 		//*************************************
 #ifdef HAVE_APPARMOR
-		else if (strcmp(argv[i], "--apparmor") == 0)
+		else if (strcmp(argv[i], "--apparmor") == 0) {
 			arg_apparmor = 1;
+			strncpy(apparmor_profile,"firejail-default",29);
+		}
+		else if (strncmp(argv[i], "--apparmor=",11) == 0) {
+			arg_apparmor = 1;
+			strncpy(apparmor_profile,argv[i]+11,29);
```

Comment:
> ```suggestion
> 			strncpy(apparmor_profile, argv[i] + 11, 29);
> ```

Many changed lines do not match the style used in the rest of the file.

For example, they are missing a space after the comma and around the operators.

Reply:
Sorry!